src/app/GUI.java

```diff
@@ -336,6 +346,39 @@ private void setupGUI() {
     lbl_moveSelectXP = new JLabel("XP");
     lbl_moveSelectXP.setBounds(339, 50, 97, 17);
     panel_moveSelect.add(lbl_moveSelectXP);
+    
+    panel_moveInventory = new JPanel();
+    frame.getContentPane().add(panel_moveInventory, "name_29070680417700");
```

Comment:
For the `CardLayout` to work, I think you need to change the stuff in the quotes to the same name as the panel.